apnsworker/src/main/scala/apnsworker/ApnsClient.scala

```diff
@@ -0,0 +1,104 @@
+package apnsworker
+
+import java.io.ByteArrayInputStream
+import java.nio.charset.StandardCharsets
+
+import _root_.models.{Notification}
+import java.sql.Timestamp
+
+import apnsworker.models.{ApnsConfig, ApnsFeedbackFailure}
+import com.turo.pushy.apns.auth.ApnsSigningKey
+import com.turo.pushy.apns.util.concurrent.{PushNotificationFuture, PushNotificationResponseListener}
+import com.turo.pushy.apns.util.{ApnsPayloadBuilder, SimpleApnsPushNotification, TokenUtil}
+import com.turo.pushy.apns.{ApnsClientBuilder, PushNotificationResponse, ApnsClient => PushyApnsClient}
+
+import scala.concurrent.ExecutionContext
+import scala.util.Try
+
+object ApnsClient {
+
+  def apply(config: ApnsConfig): Try[PushyApnsClient] = {
+    val apnsServer =
+      if (config.sendingToProdServer) ApnsClientBuilder.PRODUCTION_APNS_HOST
+      else ApnsClientBuilder.DEVELOPMENT_APNS_HOST
+
+    val signingKey = ApnsSigningKey.loadFromInputStream(
+      new ByteArrayInputStream(config.certificate.getBytes(StandardCharsets.UTF_8)),
+      config.teamId,
+      config.keyId
+    )
+
+    Try(
+      new ApnsClientBuilder()
+        .setApnsServer(apnsServer)
+        .setSigningKey(signingKey)
+        .build()
+    )
+  }
+
+  def sendNotificationE(token: String, notification: Notification)
+    (onComplete: Either[Throwable, String] => Unit)
+    (client: PushyApnsClient, config: ApnsConfig)
+    (implicit executionContext: ExecutionContext): Unit = {
+
+    val payloadBuilder = new ApnsPayloadBuilder()
+    payloadBuilder.setAlertTitle(notification.title)
+    payloadBuilder.setAlertBody(notification.message)
+
+    //      payloadBuilder.setActionButtonLabel()
+    //      payloadBuilder.setAlertSubtitle()
+    //      payloadBuilder.setBadgeNumber()
+    //      payloadBuilder.setCategoryName()
+    //      payloadBuilder.setContentAvailable()
+    //      payloadBuilder.setLaunchImageFileName()
+    //      payloadBuilder.setLocalizedActionButtonKey()
+    //      payloadBuilder.setLocalizedAlertMessage()
+    //      payloadBuilder.setLocalizedAlertSubtitle()
+    //      payloadBuilder.setLocalizedAlertTitle()
+    //      payloadBuilder.setMutableContent()
+    //      payloadBuilder.setPreferStringRepresentationForAlerts()
+    //      payloadBuilder.setShowActionButton()
+    //      payloadBuilder.setSound()
+    //      payloadBuilder.setThreadId()
+    //      payloadBuilder.setUrlArguments()
+    //      payloadBuilder.addCustomProperty()
```

Comment:
@alexduf That's the part I need help to make sure I don't forget to set any parameters and properly translate our notification model to the Pushy payload